deploy/rejected_common_passwords.txt

```diff
@@ -0,0 +1 @@
+qpcpassw0rd
```

Comment:
There should be `qpcpassw0rd` **and** `dscpassw0rd`. The latter is what downstream docs use.

But can we extract rejection logic to another PR, that will be merged few days after this one? This will give us some time to update Jenkins to use new password. Right now we would need to prepare PRs in few repos and merge them all at the same time.

Reply:
👍 I can keep this file empty (and skip the tests) for now until we're ready from the Jenkins side.

Reply:
np, commit https://github.com/quipucords/quipucords/pull/2502/commits/9ccf78871f291d3117d302b60cad9fd6ae92c8c4 adds dscpassw0rd.